CodableExample/Image.swift

```diff
@@ -31,11 +30,12 @@ class Image: Media {
     var predominantColor: UIColor
     
     required init(from skeleton: MediaSkeleton) {
-        width = skeleton.meta.width
-        height = skeleton.meta.height
-        formatExtension = skeleton.meta.format
+        let meta = skeleton.meta as! ImageMetaSkeleton
```

Comment:
This is the only thing I am somewhat worried about. But I honestly don't think it will cause issues or crash. If it does it would be programmer error.